registry/Scripts/script.sh

```diff
@@ -301,16 +322,29 @@ if [ $ENABLE_VALIDATIONS == "true" ]; then
     if [ $? -ne 0 ]; then
         exit $ERR_REGISTRY_LOGIN_FAILED
     fi
-    docker tag registry:${REGISTRY_IMAGE_TAG} localhost:443/registry:${REGISTRY_IMAGE_TAG}
-    docker push localhost:443/registry:${REGISTRY_IMAGE_TAG}
+    cat <<EOF >> Dockerfile
+    FROM registry:${REGISTRY_IMAGE_TAG}
+    RUN touch registry 
+EOF
+
+    TEST_IMAGE_NAME="localhost:443/testbuild"
```

Comment:
missing tag? `localhost:443/testbuild:123`

Reply:
that's ok as tag will become latest.